chat-box/style.css

```
@@ -0,0 +1,299 @@
+* {
+    padding: 0;
+    margin: 0;
+    font-family: 'Poppins', sans-serif;
+}
+
+.wrapper-chat-box {
+    display: flex;
+
+    width: 630px;
+    height: 650px;
+    margin: 20px auto;
+    
+}
+
+.list-friend {
+    width: 230px;
+    height: 100%;
+
+    background-color: #454753;
+}
+
+.box-chat {
+    width: 400px;
+    height: 100%;
+
+    background-color: #fff;
+}
+
+.search-form {
+    position: relative;
+    margin-top: 17px;
+    margin-left: 17px;
+
+    width: 167px;
+    margin-bottom: 35px;
+}
+.search-form > .fa-search {
+    position: absolute;
+    color:#F4F5FA;
+    line-height: 35px;
+    font-size: 12px;
+    right: 8px;
+    top: 0;
+}
+.search-form input {
+    height: 35px;
+    background: #696C73;
+    border-style: none;
+    padding-left: 10px ;
+
+    border-radius: 2px;
+}
+
+.users>ul>li {
+    list-style: none;
+}
+
+.user-status {
+    display: flex;
+
+    margin-left: 17px;
+    margin-bottom: 15px;
+
+    width: 167px;
+    height: 40px;
+}
+.offline .fa-circle {
+    color: #DE8C66!important;
+}
+.avt-image {
+    height: 40px;
+    width: 40px;
+    border-radius: 50%;
+    border : 3px solid #696C73;
+    background-color: lightslategray;
+    box-sizing: border-box;
+    object-fit: cover;
+    overflow:hidden;
+}
+.user-information>.avt-image {
+    border-color: #91BA78;
+}
+.avt-image>img {
```

dấu `>` thêm dấu cách vào
```suggestion
.avt-image > img {
```